README.md

```diff
@@ -1,35 +1,35 @@
-# K3PO for .NET
-
-[![Build Status][build-status-image]][build-status]
-[![Issue Stats][pull-requests-image]][pull-requests]
-[![Issue Stats][issues-closed-image]][issues-closed]
-
-[build-status-image]: https://ci.appveyor.com/api/projects/status/erbxtvrxc3j59nqs/branch/develop?svg=true
-[build-status]: https://ci.appveyor.com/project/jfallows/k3po-dotnet/branch/develop
-[pull-requests-image]: http://www.issuestats.com/github/k3po/k3po.dotnet/badge/pr
-[pull-requests]: http://www.issuestats.com/github/k3po/k3po.dotnet
-[issues-closed-image]: http://www.issuestats.com/github/k3po/k3po.dotnet/badge/issue
-[issues-closed]: http://www.issuestats.com/github/k3po/k3po.dotnet
-
-- .Net K3PO Testing Framework
+# K3PO for .NET
+
+[![Build Status][build-status-image]][build-status]
+[![Issue Stats][pull-requests-image]][pull-requests]
+[![Issue Stats][issues-closed-image]][issues-closed]
+
+[build-status-image]: https://ci.appveyor.com/api/projects/status/erbxtvrxc3j59nqs/branch/develop?svg=true
+[build-status]: https://ci.appveyor.com/project/jfallows/k3po-dotnet/branch/develop
+[pull-requests-image]: http://www.issuestats.com/github/k3po/k3po.dotnet/badge/pr
+[pull-requests]: http://www.issuestats.com/github/k3po/k3po.dotnet
+[issues-closed-image]: http://www.issuestats.com/github/k3po/k3po.dotnet/badge/issue
+[issues-closed]: http://www.issuestats.com/github/k3po/k3po.dotnet
+
+- .Net K3PO Testing Framework
 - NUnit Extension for K3PO Testing
 
-
-### Requirement
-
-
+
+### Requirement
+
+
 - Visual Studio 2013. The community edition can be downloaded from [here](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx)
-- NUnit 2.6.3
+- NUnit 3.0.1
+
+### Steps to run the sample test
 
-### Steps to run the sample test
-
 - Open K3PO.sln in Visual Studio
-- Build the solution
+- Build the solution.
+    - If the build failed because missing nunit.framework. make sure [Visual Studio is configured to 'Allow NuGet to download missing packages'](https://docs.nuget.org/consume/package-restore)

```

Comment:
There are typos in this line. Perhaps this line should be changed as follows:

    - If the build failed because of missing `nunit.framework`, make sure Visual Studio is configured to [Allow NuGet to download missing packages](https://docs.nuget.org/consume/package-restore)